Flat/src/PandaAnalyzer.cc

```diff
@@ -442,6 +442,27 @@ void PandaAnalyzer::SetDataDir(const char *s)
     if (DEBUG) PDebug("PandaAnalyzer::SetDataDir","Loaded btag SFs");
   }
 
+  // bjet regression
+  if (analysis->bjetRegression){
```

Comment:
I think there needs to be a separate flag for *applying* vs *training* the b-jet regression. `bjetRegression` currently turns on clustering of gen jets with neutrinos, which is expensive and only needed in the latter case.